deploy/vagrant/vbox/buildbox.sh

```diff
@@ -2,9 +2,8 @@
 
 OLD_BOX=$(vagrant box list | grep tachyon-dev | cut -d ' ' -f1)
 if [[ "$OLD_BOX" != '' ]]; then
- echo "$OLD_BOX exists"
- echo "if you want to remove $OLD_BOX, use command: vagrant box remove $OLD_BOX"
- exit 0
+  echo "Tachyon base image $OLD_BOX exists."
```

Comment:
I think it may be worth including the help for removing the old box here still. The user may not run destroy and find it there.